README.md

```diff
@@ -50,6 +50,34 @@ Documentation for older versions or branches of s2n-tls can be generated locally
 
 Doxygen installation instructions are available at the [Doxygen](https://doxygen.nl/download.html) webpage.
 
+## Platform Support
+
+We’ve listed the distributions and platforms under two tiers: Tier 1 platforms are guaranteed to build, run, and pass tests in CI. Tier 2 platforms are guaranteed to build and we'll address issues opened against them, but they aren't currently running in our CI and are not actively reviewed with every commit. If you use a platform not listed below and would like to request (or help!) add it to our CI, please open an issue for discussion.
+
+### Tier 1
+
+|Distribution in CI	                                    |Platforms |
+|-------------------------------------------------------|---------|
+|Ubuntu18/22/24**                                       | x86_64  |
+|Ubuntu22                                               | i686    |
```

Comment:
From offline discussion: It's odd to include Ubuntu22 in two different rows. You should probably just list both platforms on the second row.

And do other recent Ubuntu versions not support aarch64 too? Should that appear in Tier 2?